### DISCUSSION TITLE
Filtering in resource-columns?

### DISCUSSION THREAD

**Discussion by SnorreSelmer** [aptakube/aptakube#327] [Q&A] [answered]
Is it possible to filter out certain containers from the resource-requests/limits bar in (for example) Pods-view?

We run Istio on our clusters for service-meshing, so the Pods-view shows the resources of both the primary container and the istio-proxy sidecar container. If I could do a regex exclude like `!~"istio-.*"` I could get an easier overview for confirming that resources are tuned appropriately.

**Comment by goenning** [comment, marked as answer]
Not currently. But I’m working on a “Workloads Overview” page to highlight things that users should be aware of, and this might be good thing to add as well.

how do you “confirm that resources are tuned appropriately”? Are you basically looking at usage vs limit?

**Reply by SnorreSelmer** [reply]
Yeah.

For our setup I have to click on each pod for a namespace to look at the resource-usage of the container in question (for example grafana-mimir-gateway), make a note of CPU and memory usage, and then set requests accordingly in the grafana-mimir values-file.

If I take the values directly from the Pods-page, then I get resources listed as "istio-consumption + gateway-consumption" of "istio-requests + gateway-requests". If they both request 128Mi, and istio-proxy overconsumes, and gateway underconsumes, then it's easy to get the impression that requests are fine, when gateway really can use a smaller request. (Istio isn't my responsibility, and is controlled elsewhere, so I don't care about it.)

This is why I'd like an option to remove istio-related containers so I can get more representative resource-consumption for the main application. I set requests based on the "average" consumption, nothing fancy (yet). This is a temporary measure until we can get a working automatic resource-scheduler up and running.
